tests/unit/modules/test_win_service.py

```diff
@@ -187,6 +187,20 @@ def test_stop_not_running(self):
                 patch.object(win_service, '_status_wait', mock_info):
             self.assertTrue(win_service.stop('spongebob'))
 
+    @skipIf(not WINAPI, 'pywintypes not available')
+    def test_stop_refused(self):
+        '''
+        Test stopping a service, but Windows refuses to stop it
+        '''
+        mock_error = MagicMock(
+            side_effect=pywintypes.error(1061,
```

Comment:
```suggestion
            side_effect=pywintypes.error(1062,
```
This just has the wrong error code for the win_service:
```
    try:
        win32serviceutil.StopService(name)
    except pywintypes.error as exc:
        if exc.winerror != 1062:
            raise CommandExecutionError(
                'Failed To Stop {0}: {1}'.format(name, exc.strerror))
        log.debug('Service "{0}" is not running'.format(name))
```